Link_sinFade_Library.ino

```diff
@@ -238,7 +238,7 @@ void HSVwriteToLEDs(float someHSV[], int someZone) {  // <<< Includes whiteBalan
   RGB_from_hsv[1] = RGB_from_hsv[1] * GrnScaler / 100.0;  // GrnScaler = 77    //  67.0
   RGB_from_hsv[2] = RGB_from_hsv[2] * BluScaler / 100.0;  // BluScaler = 57    //  37.0
   for (byte led = 0; led < 3; led++) {
-    analogWrite(LunitZones[someZone][led], RGB_from_hsv[led]);  // Display color step.
+    PWM.set(LunitZones[someZone][led], RGB_from_hsv[led]);  // Display color step.
```

Comment:
That's it. The pattern is:
```
PWM_NativePin_MEGA PWM;
...
void somewhere() {
  PWM.set( physical_pin, pwm_value);
}
```

Reply:
I see the parallel between analogWrite() and PWM.set(). The range of values sent to pins by HSVwriteToLEDs() is 0 to 255. The purpose of all this is to increase this range, something like 0 to 65,535. 
HSVehtoRGB() currently produces an RGB array for the brightest white in the form [255, 255, 255]. This function will have to be altered to make bright white be [65,535, 65,535, 65,535]..........  Is this true?

Reply:
pr #4 does that